in_toto/runlib.py

```diff
@@ -149,6 +149,11 @@ def record_artifacts_as_dict(artifacts, exclude_patterns=None,
             endings before hashing the content of the passed files, for
             cross-platform support.
 
+    lstrip_paths: (optional)
+            If a prefix path is passed, the prefix is left stripped from
+            the path of every artifact that contains the prefix. Currently,
```

Comment:
I'd drop the `currently, the prefix is a single path.` as it just clutters the description (let's mention that more can be provided in the future instead).